rootfs/api/models/certificate.py

```diff
@@ -106,14 +113,23 @@ def domains(self):
     def __str__(self):
         return self.name
 
-    def _get_certificate(self):
+    def _load_certificate(self):
         try:
```

Comment:
Wouldn't this be a bit more DRY if it just called `validate_private_key(self.certificate)` from above and re-raised `ValidationError` as `SuspiciousOperation`?